server/lib/cleanup/languages/fr.js

```diff
@@ -49,9 +51,134 @@ function clean(sentences) {
+	  .replace(/(^|\s)IV(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' quatrième siècle ')
+	  .replace(/(^|\s)V(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' cinquième siècle ')
+	  .replace(/(^|\s)VI(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' sixième siècle ')
+	  .replace(/(^|\s)VII(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' septième siècle ')
+	  .replace(/(^|\s)VIII(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' huitième siècle ')
+	  .replace(/(^|\s)(VIIII|IX)(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' neuvième siècle ')
+	  .replace(/(^|\s)X(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' dixième siècle ')
+	  .replace(/(^|\s)XI(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' onzième siècle ')
+	  .replace(/(^|\s)XII(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' douxième siècle ')
+	  .replace(/(^|\s)XIII(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' treizième siècle ')
+	  .replace(/(^|\s)(XIIII|XIV)(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' quatorzième siècle ')
+	  .replace(/(^|\s)XV(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' quinzième siècle ')
+	  .replace(/(^|\s)XVI(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' seixième siècle ')
+	  .replace(/(^|\s)XVII(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' dix-septième siècle ')
+	  .replace(/(^|\s)XVIII(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' dix-huitième siècle ')
+	  .replace(/(^|\s)(XIX|XVIIII)(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' dix_neuvième siècle ')
+	  .replace(/(^|\s)XX(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' vingtième siècle ')
+	  .replace(/(^|\s)XXI(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' vingt-et-unième siècle ')
+	  .replace(/(^|\s)XXII(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' vingt-deuxième siècle ')
+      
+     //roman numerals.
+	  .replace(/(^|\s)I(\s|\.|,|\?|!|$)/g, ' premier ') //translated as 'first'. We considere that it's encountered after chapter (Chapitre I => 'chapitre premier'). Work also with names (Charles I => 'Charles premier')
+	  .replace(/(^|\s)II(\s|\.|,|\?|!|$)/g, ' deux ')
+	  .replace(/(^|\s)III(\s|\.|,|\?|!|$)/g, ' trois ')
+	  .replace(/(^|\s)IV(\s|\.|,|\?|!|$)/g, ' quatre ')
+	  .replace(/(^|\s)V(\s|\.|,|\?|!|$)/g, ' cinq ')
+	  .replace(/(^|\s)VI(\s|\.|,|\?|!|$)/g, ' six ')
+	  .replace(/(^|\s)VII(\s|\.|,|\?|!|$)/g, ' sept ')
+	  .replace(/(^|\s)VIII(\s|\.|,|\?|!|$)/g, ' huit ')
+	  .replace(/(^|\s)(VIIII|IX)(\s|\.|,|\?|!|$)/g, ' neuf ')
+	  .replace(/(^|\s)X(\s|\.|,|\?|!|$)/g, ' dix ')
+	  .replace(/(^|\s)XI(\s|\.|,|\?|!|$)/g, ' onze ')
+	  .replace(/(^|\s)XII(\s|\.|,|\?|!|$)/g, ' douze ')
+	  .replace(/(^|\s)XIII(\s|\.|,|\?|!|$)/g, ' treize ')
+	  .replace(/(^|\s)(XIIII|XIV)(\s|\.|,|\?|!|$)/g, ' quatorze ')
+	  .replace(/(^|\s)XV(\s|\.|,|\?|!|$)/g, ' quinze ')
+	  .replace(/(^|\s)XVI(\s|\.|,|\?|!|$)/g, ' seize ')
+	  .replace(/(^|\s)XVII(\s|\.|,|\?|!|$)/g, ' dix-sept ')
+	  .replace(/(^|\s)XVIII(\s|\.|,|\?|!|$)/g, ' dix-huit ')
+	  .replace(/(^|\s)(XIX|XVIIII)(\s|\.|,|\?|!|$)/g, ' dix-neuf ')
+	  .replace(/(^|\s)XX(\s|\.|,|\?|!|$)/g, ' vingt ')
+ 	  .replace(/(^|\s)XXI(\s|\.|,|\?|!|$)/g, ' vingt-et-un ')
+ 	  .replace(/(^|\s)XXII(\s|\.|,|\?|!|$)/g, ' vingt-deux ')
```

Comment:
I think we can remove all of the roman numerals, as they are not allowed from the beginning (see the abbreviation pattern in the validation file).